File: pgm/ABC_C#/ABC_160_B.cs
using System;
// using System.Collections.Generic;

class Program
{
    static void Main()
    {
        int X = int.Parse(Console.ReadLine());
        int rm = X % 500;
        Console.WriteLine(X / 500 * 1000 + rm / 5 * 5);
    }
}

File: pgm/ABC_Visual_Basic/ABC_160_B.vb
' Imports System.Collections.Generic
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim X As Integer = Console.ReadLine()
        Dim rm As Integer = X Mod 500
        Console.WriteLine(Math.Floor(X / 500) * 1000 + Math.Floor(rm / 5) * 5)
    End Sub
End Class
